pkg/inspector/image-inspector.go

```diff
@@ -649,3 +606,79 @@ func createOutputDir(dirName string, tempName string) (string, error) {
 	}
 	return dirName, nil
 }
+
+// acquireImage returns error, the docker image that was acquired, the image ID,
+// the container ID that the image was acquired from, and iiapi.FilesFilter for this image.
+func (i *defaultImageInspector) acquireImage(client DockerRuntimeClient) (error, docker.Image, string, string, iiapi.FilesFilter) {
```

Comment:
@enoodle when you return such a large number of items you may want to consider an object approach, e.g. either you make those attributes of the relevant `defaultImageInspector` class (if it makes sense), or you return an object of a new class.